config/config.exs

```diff
@@ -39,3 +39,18 @@ config :lambda_ethereum_consensus, BeaconApi.Endpoint,
 config :lambda_ethereum_consensus, ChainSpec,
   config: MinimalConfig,
   genesis_validators_root: <<0::256>>
+
+# Configure sentry logger handler
+# To enable sentry, set the SENTRY_DSN environment variable to the DSN of your sentry project
+config :sentry,
+  dsn: System.get_env("SENTRY_DSN"),
+  environment_name: Mix.env(),
+  enable_source_code_context: true,
+  root_source_code_paths: [File.cwd!()]
+
+config :lambda_ethereum_consensus, :logger, [
+  {:handler, :sentry_handler, Sentry.LoggerHandler,
+   %{
+     config: %{metadata: [:file, :line]}
```

Comment:
should we add additional metadata like block hash or slot?

Reply:
Sure!

Reply:
I added the slot one (I also found we weren't capturing normal ERROR messages 😅), but the root one doesn't seem to work. I guess since it's a raw binary the handler doesn't know how to format it.